install.sh

```diff
@@ -129,6 +134,27 @@ main () {
             --no-update-shell-config)
                 should_update_shell_config="n"
                 ;;
+            --shell-config)
+                if [ "$#" -eq "0" ]; then
+                    error "--shell-config must be passed an argument"
+                fi
+                shell_config="$1"
+                shift
+                ;;
+            --shell)
+                if [ "$#" -eq "0" ]; then
+                    error "--shell must be passed an argument"
```

Comment:
I was a bit confused about this, thinking that this makes `--shell` a mandatory flag. I talked with @gridbugs offline, and it turns out I had misunderstood it. All good with it!